packages/polymath-issuer/src/pages/compliance/components/ApprovalTable.js

```diff
@@ -0,0 +1,238 @@
+// @flow
+
+import React, { Component } from 'react';
+import { DataTable, Icon } from 'carbon-components-react';
+import { Button, addressShortifier } from '@polymathnetwork/ui';
+import ApprovalModal from './ApprovalModal';
+import { connect } from 'react-redux';
+import {
+  removeApprovalFromApprovals,
+  editApproval,
+} from '../../../actions/compliance';
+import moment from 'moment';
+const {
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableBody,
+  TableCell,
+  TableHeader,
+  TableToolbar,
+  TableToolbarContent,
+} = DataTable;
+
+const columns = [
+  {
+    header: 'From Investor Wallet Address',
+    key: 'fromAddress',
+  },
+  {
+    header: 'To Investor Wallet Address',
+    key: 'toAddress',
+  },
+  {
+    header: 'Approval Expiry',
+    key: 'expiry',
+  },
+  {
+    header: 'Unique ID (TxHash) of Approval',
```

Comment:
I don't see this `Unique ID` column displayed. I'm not even sure if it's necessary TBH

<img width="1182" alt="Screen Shot 2019-11-08 at 4 15 54 AM" src="https://user-images.githubusercontent.com/1994818/68476020-7d9c8a80-01de-11ea-8406-7a432f676a4a.png">

Reply:
It's hidden so the data is available when you click Approved token transfer. Goal is to not have to search an array every single time